### PR TITLE
feat(b-col): restore `.offset-*` col classes + new b-container and b-row components 🍾🍻🎉

### PR DESCRIPTION
**Pull request by alexsasharegan**
Per our beta ship list (#747) and BS4.beta.2: twbs/bootstrap#23278, let's bring back the offset classes!

### CODE REVIEW

**Comment by tmorehouse** [comment]
Added checkbox to BSV4.beta shiplist #747

File: docs/components/layout/README.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+While containers can be nested, most layouts do not require a nested container.
+
+**Fixed width container, based on viewport breakpoints:**
+```html
+<b-container>
+  <!-- Content here -->
+</b-container>
+```
+
+**Fluid container which is always 100% width, regardless of viewport brekpoint:**
```
Comment (alexsasharegan) [inline docs/components/layout/README.md]: ⌨️💥

File: docs/components/layout/README.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+`<b-col>` Must be placed inside a `<b-row>` component, or an element (such as a `<div>`)
+that has the class `row` applied to it, or - in the case of [forms](/docs/components/form) -
+inside a `<b-form-row>` component to obtain columns with more compact margins.
+
+`<b-col>` provides many optional props for controling width and offest at
+various viewport breakpoints.
+
+**Note:** _The `offest-*` props will work once Bootstrap V4.0.0.beta.2 is released.
+The current V4.0.0.beta.1 CSS does not include the required `.offest-*` classes._
+
```
Comment (alexsasharegan) [inline docs/components/layout/README.md]: 👍🏼

File: docs/components/layout/README.md (hunk 1 above)
Comment (alexsasharegan) [reply docs/components/layout/README.md]: This is the only typo I found

Comment (tmorehouse) [reply docs/components/layout/README.md]: That's good record for me, lol

File: docs/nuxt/components/nav.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <b-navbar toggleable class="navbar-dark bd-navbar" fixed="top">
+    <b-navbar toggleable="md" class="navbar-dark bd-navbar" fixed="top">
```
Comment (tmorehouse) [inline docs/nuxt/components/nav.vue]: 👍

**Pull request merged**